iceoryx_hoofs/source/posix_wrapper/named_pipe.cpp

```diff
@@ -138,11 +175,11 @@ NamedPipe::~NamedPipe() noexcept
     IOX_DISCARD_RESULT(destroy());
 }
 
-IpcChannelName_t NamedPipe::convertName(const IpcChannelName_t& name) noexcept
+template <typename Prefix>
+IpcChannelName_t NamedPipe::convertName(const Prefix& p, const IpcChannelName_t& name) noexcept
 {
-    return IpcChannelName_t(
-        cxx::TruncateToCapacity,
-        cxx::concatenate(NAMED_PIPE_PREFIX, (name.c_str()[0] == '/') ? *name.substr(1) : name).c_str());
+    return IpcChannelName_t(cxx::TruncateToCapacity,
+                            cxx::concatenate(p, (name.c_str()[0] == '/') ? *name.substr(1) : name).c_str());
```

Comment:
I've seen this already elsewhere. Does this qualify for a `concatenatePath` helper function?

Reply:
Yeah you know it from the previous PR ;) ... We use `cxx::concatenate` only in this file and in the unit tests for the string